kore/src/Kore/Attribute/Axiom.hs

```diff
@@ -200,12 +205,12 @@ instance From symbol SymbolOrAlias => From (Axiom symbol variable) Attributes wh
             , from . owise
             ]
 
-instance SQL.Column (Axiom SymbolOrAlias variable) where
+instance SQL.Column (Axiom SymbolOrAlias Variable) where
```

Comment:
Why does this change? (I think it's fine, I'm just not sure why.)

Reply:
it's because the From instance for Concrete needs a `Variable`, because it needs `concreteAttribute`, which needs `UnifiedVariable Variable`